Client/Frontend/Home/TabsPanel.swift

```diff
@@ -79,16 +53,8 @@ class TabsPanel: UITableViewController {
         return cell
     }
 
-    override func tableView(tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return 42
-    }
-
-    override func tableView(tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
-        return 0
-    }
-
     override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-        let view = tableView.dequeueReusableHeaderFooterViewWithIdentifier(TABS_HEADER_IDENTIFIER) as? UIView
+        let view = super.tableView(tableView, viewForHeaderInSection: section) as UIView!
```

Comment:
No need for the <code>as UIView!</code> here -- you're checking the optional below anyway.